airbyte-integrations/connectors/destination-clickhouse/src/test-integration/java/io/airbyte/integrations/destination/clickhouse/ClickhouseDestinationAcceptanceTest.java

```diff
@@ -0,0 +1,202 @@
+/*
+ * Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.destination.clickhouse;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.google.common.collect.ImmutableMap;
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.db.Databases;
+import io.airbyte.db.jdbc.JdbcDatabase;
+import io.airbyte.integrations.base.JavaBaseConstants;
+import io.airbyte.integrations.destination.ExtendedNameTransformer;
+import io.airbyte.integrations.standardtest.destination.DestinationAcceptanceTest;
+import io.airbyte.workers.process.ProcessFactory;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.jooq.JSONFormat;
+import org.jooq.JSONFormat.RecordFormat;
+import org.junit.jupiter.api.Disabled;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.ClickHouseContainer;
+import org.testcontainers.containers.Network;
+
+public class ClickhouseDestinationAcceptanceTest extends DestinationAcceptanceTest {
+
+  private static final String DB_NAME = "default";
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(ClickhouseDestinationAcceptanceTest.class);
+  private static final JSONFormat JSON_FORMAT = new JSONFormat().recordFormat(RecordFormat.OBJECT);
+  private final ExtendedNameTransformer namingResolver = new ExtendedNameTransformer();
+
+  private ClickHouseContainer db;
+  private Network network;
+
+  private ProcessFactory processFactory;
+  private TestDestinationEnv testEnv;
+
+  private boolean useNativePort = false;
+
+  @Override
+  protected String getImageName() {
+    return "airbyte/destination-clickhouse:dev";
+  }
+
+  @Override
+  protected boolean supportsNormalization() {
+    return true;
+  }
+
+  @Override
+  protected boolean supportsDBT() {
+    return true;
+  }
+
+  @Override
+  protected boolean implementsNamespaces() {
+    return true;
+  }
+
+  @Override
+  protected String getDefaultSchema(final JsonNode config) {
+    if (config.get("database") == null) {
+      return null;
+    }
+    return config.get("database").asText();
+  }
+
+  @Override
+  protected JsonNode getConfig() {
+    // ClickHouse official JDBC driver use HTTP protocal, its default port is 8123
+    // dbt clickhouse adapter use native protocal, its default port is 9000
+    return Jsons.jsonNode(ImmutableMap.builder()
+        .put("host", db.getHost())
+        .put("port", useNativePort ? db.getMappedPort(ClickHouseContainer.NATIVE_PORT) : db.getFirstMappedPort())
+        .put("database", DB_NAME)
+        .put("username", db.getUsername())
+        .put("password", db.getPassword())
+        .put("schema", DB_NAME)
+        .build());
+  }
+
+  @Override
+  protected JsonNode getFailCheckConfig() {
+    String ipAddress = db.getContainerInfo().getNetworkSettings().getIpAddress();
```

Comment:
What purpose of this variable?

Reply:
that's a leftover when I was struggling with container's connectivity doing check test, it can be removed. See https://github.com/testcontainers/testcontainers-java/issues/452.